website/content/docs/connect/config-entries/http-route.mdx

```diff
@@ -733,9 +845,134 @@ Specifies rule for rewriting the URL of incoming requests.
+| `reset`              | Consul retries the request when the upstream does not respond at all.                                    |
+| `connect-failure`    | Consul retries the request when the connection to the upstream fails.                                    |
+| `envoy-ratelimited`  | Consul retries the request when the header `x-envoy-ratelimited` is present.                             |
+| `retriable-4xx`      | Consul retries the request when the upstream responds with a retriable 4xx code.                         |
+| `refused-stream`     | Consul retries the request when the upstream resets the stream with a `REFUSED_STREAM` error code.       |
+| `cancelled`          | Consul retries the request when the gRPC status code in the response headers is `cancelled`.             |
+| `deadline-exceeded`  | Consul retries the request when the gRPC status code in the response headers is `deadline-exceeded`.     |
+| `internal`           | Consul retries the request when the gRPC status code in the response headers is `internal`.              |
+| <nobr>`resource-exhausted`</nobr> | Consul retries the request when the gRPC status code in the response headers is `resource-exhausted`.    |
+| `unavailable`        | Consul retries the request when the gRPC status code in the response headers is `unavailable`.           |
+
+#### Values
+
+- Default: None
+- Data type: List of strings. Strings must match one of the following values:
+
+  - `5xx`
+  - `gateway-error`
+  - `reset`
+  - `connect-failure`
+  - `envoy-ratelimited`
+  - `retriable-4xx`
+  - `refused-stream`
+  - `cancelled`
+  - `deadline-exceeded`
+  - `internal`
+  - `resource-exhausted`
+  - `unavailable`
+
+### ``Rules[].Services[].Filters{}.RetryFilters{}.RetryOnStatusCodes`
+
+Specifies a list of integers for [HTTP response status codes](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status) that trigger a retry request.
+
+#### Values
+
+- Default: None
+- Data type: List of integers
+
+### `Rules[].Service[].Filters{}.TimeoutFilter`
+
+Specifies timeout settings for routes from an API gateway listener to the destination service in Consul service mesh. 
+
+#### Values
+
+- Default: None
+- Data type: Map
+
+The following table describes the settings you can configure in the `TimeoutFilter` map:
+
+| Parameter | Description | Data type | Default |
+| ---       | ---         | ---       | ---     |
+| `IdleTimeout` | Specifies the total amount of time permitted for the request stream to be idle. | Integer | `0` |
+| `RequestTimeout` | Specifies the total amount of time in nanoseconds, including retry attempts, Consul permits for the entire downstream request to be processed. | Integer | `0` |
+
+
 ## Examples
 
-The following examples demonstrate common HTTP route configuration patterns for specific use cases.
+The following examples demonstrates how to set an idle timeout and a request timeout. 
```

Comment:
This looks to me like the example for the k8s got accidentally added to the example for VM config entry?